### PR TITLE
bsunanda:Run2-hcx84 Adding missing components fro FrontEndMap

### PR DESCRIPTION
**Pull request by bsunanda**
Several important changes were missing in previous version to import FrontEndMap using the condb_import macro and also to save this to db

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by abdoulline** [comment]
I'd like to ask Giacomo and Marco to approve 
 @ggovi, @mmusich

**Comment by mmusich** [comment]
@bsunanda please put some description of the PR. This will greatly expedite the prioritization of the reviews.

**Comment by mmusich** [comment]
+1
I gather this fixes issues left over in https://github.com/cms-sw/cmssw/pull/14456.

File: CondCore/Utilities/src/CondDBFetch.cc [hunk 1]
```diff
@@ -140,9 +140,10 @@ namespace cond {
       FETCH_PAYLOAD_CASE( HBHENegativeEFilter )
       FETCH_PAYLOAD_CASE( HcalChannelQuality )
       FETCH_PAYLOAD_CASE( HcalCholeskyMatrices )
+      FETCH_PAYLOAD_CASE( HcalDcsValues )
       FETCH_PAYLOAD_CASE( HcalElectronicsMap )
       FETCH_PAYLOAD_CASE( HcalFlagHFDigiTimeParams )
-      FETCH_PAYLOAD_CASE( HcalDcsValues )
+      FETCH_PAYLOAD_CASE( HcalFrontEndMap )
       FETCH_PAYLOAD_CASE( HcalGains )
```
Comment (mmusich) [inline CondCore/Utilities/src/CondDBFetch.cc]: @bsunanda thanks, this will allow to use the condb_import macro on the new CondFormat

**Comment by ggovi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**